fs/vfs/fs_lock.c

```diff
@@ -181,7 +184,6 @@ static int file_lock_normalize(FAR struct file *filep,
     }
 
   out->l_whence = SEEK_SET;
-  out->l_pid = getpid();
```

Comment:
Why was PID removed?

Reply:
The reason for removing the PID from normalize is that both setlk and getlk reuse this interface to convert the flock once.
For getlk, in some test scenarios it is not expected to change the contents of a user-inputted flock when it cannot find a conflicting lock, so getpid is taken out separately and placed in the setlk process.

Reply:
@acassis do you have more comment?